OpenSim/Simulation/Model/Model.cpp

```diff
@@ -806,6 +807,10 @@ void Model::extendAddToSystem(SimTK::MultibodySystem& system) const
 {
     Model *mutableThis = const_cast<Model *>(this);
 
+    //Analyses are not Components so add them after legit 
+    //Components have been wired-up correctly.
+    mutableThis->updAnalysisSet().setModel(*mutableThis);
+
```

Comment:
connectToModel seemed like a fine place for this; why the move? Not really important; feel free to ignore this.